### PR TITLE
Update readme 20181219

### PR DESCRIPTION
**Pull request by keigohtr**
## What is this PR for?

To accelerate expanding Rekcurd, we need to update our installation manual.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,102 @@
...
-- [Drucker-example](https://github.com/drucker/drucker-example): Example of how to use drucker.
+- [Rekcurd](https://github.com/rekcurd/drucker): Project for serving ML module. This is a gRPC micro-framework and it can be used like [Flask](http://flask.pocoo.org/).
+- [Rekcurd-dashboard](https://github.com/rekcurd/drucker-dashboard): Project for managing ML model and deploying ML module. Any Rekcurd service is manageable. It can deploy the Rekcurd service to Kubernetes cluster.
+- [Rekcurd-client](https://github.com/rekcurd/drucker-client): Project for integrating ML module. Any Rekcurd service is connectable. It can connect the Rekcurd service on Kubernetes.
+
+### Sub components
+- [dockerfiles](https://github.com/rekcurd/dockerfiles): Project for Rekcurd's container image. Since it works with [Docker Hub](https://hub.docker.com/r/rekcurd/rekcurd), you can pull image via `docker pull`.
+- [drucker-grpc-proto](https://github.com/rekcurd/drucker-grpc-proto): Project for Rekcurd's gRPC spec.
+- [Drucker-example](https://github.com/rekcurd/drucker-example): Example project. This sample turns a scikit-learn Linear SVC algorithm into a Rekcurd service.
+
```
Comment (yuki-mt) [inline README.md]: Drucker-example => Rekcurd-example?
drucker-grpc-proto => reckurd-grpc-proto?

If they will be modified when repository name is changed, it is good for now.

Comment (keigohtr) [reply README.md]: > If they will be modified when repository name is changed, it is good for now.

Yes we plan to.

**Comment by keigohtr** [comment]
@yuki-mt Thank you!

> I think we need "Get Started" documents.

Is "Installation" not a "Get Started"?

**Comment by yuki-mt** [comment]
@keigohtr 
[The document](https://github.com/rekcurd/drucker-parent/blob/fix/update-readme-20181219/docs/README.md) is good enough as the fundamental one, but for me, "Get Started" documents should be more specific to basic use case and to users.

For example,

- What to do after setting up? (e.g. deploying sample service , sending sample request, etc...)
- What should I do to prepare Rekcurd environment as an infrastructure engineer? (The current document already covers this one, but it does not mention that it is specific for infrastructure engineers)
- What should I do to run Rekcurd service as an ML engineer?

**Comment by keigohtr** [comment]
@yuki-mt OK I see.
We need to improve our doc more and more.
(Maybe the next PR...)

**Comment by keigohtr** [comment]
Thank you!

**Pull request merged**